templates/source/access-points.html

```diff
@@ -117,8 +117,17 @@
                 </div>
             </div>
             <div class="form-group">
-                <div class="col-sm-12">
+                <div class="col-sm-12 text-right">
                     <br />
+                    <button type="button" class="btn btn-danger cancel-button">
+                      {% if accesspoint.uuid %}
+                        <a href="{% url 'view-access-point' accesspoint.uuid %}">
+                      {% else %}
+                        <a href="{% url 'add-access-point' source.uuid %}">
+                      {% endif %}
+                        {% trans "Cancel" %}  <i class="fas fa-times"></i>
+                      </a>
+                    </button>
                     <button type="submit" class="btn btn-success pull-right">{% trans "Save" %}  <i class="fa fa-check"></i></button>
```

Comment:
The `pull-right` class on this button seems to be causing these buttons to be smushed together:

<img width="861" alt="Screen Shot 2019-08-08 at 10 17 39 AM" src="https://user-images.githubusercontent.com/14170650/62715272-c8cbb500-b9c5-11e9-9324-b769a7da6963.png">

If we remove it we should get the right spacing.